blenderproc/python/types/MeshObjectUtility.py

```diff
@@ -337,6 +337,32 @@ def update_from_bmesh(self, bm: bmesh.types.BMesh, free_bm_mesh=True) -> bmesh.t
         # Make sure the mesh is updated
         self.get_mesh().update()
 
+    def join_with_other_objects(self, objects: List[Self]):
+        """
+            Joins the given list of objects with this object.
```

Comment:
We should also note here that the given object references become invalid after the join operation